src/network/netdev/xfrm.c

```diff
@@ -0,0 +1,81 @@
+/* SPDX-License-Identifier: LGPL-2.1+ */
+
+#include <net/if.h>
+
+#include "netdev/xfrm.h"
+
+static int xfrm_config_verify(
+    NetDev *netdev,
+    const char *filename
+) {
```

Comment:
weird indentation, we usually use double indentation for arguments, i.e. 16ch in this case. Please follow coding style

Reply:
(here and everywhere: https://systemd.io/CODING_STYLE#formatting)